.eslintrc

```diff
@@ -1,27 +1,25 @@
 {
-  "env": {
-    "browser": true,
-    "es2020": true
-  },
-  "extends": [
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:react/jsx-runtime",
-    "plugin:react-hooks/recommended"
-  ],
-  "parserOptions": {
-    "ecmaVersion": "latest",
-    "sourceType": "module"
-  },
-  "settings": {
-    "react": {
-      "version": "18.2"
-    }
-  },
-  "plugins": [
-    "react-refresh"
-  ],
-  "rules": {
-    "react-refresh/only-export-components": "warn"
-  }
+	"env": {
```

Comment:
Remove this from PR and avoid doing this. If it's just spacing and not needed. I think this is due to your default linter on vsCode